src/components/bs5/quickexit/quickexit.scss

```diff
@@ -0,0 +1,178 @@
+/*
+ * Mobile layout
+ */
+.qld-quick-exit {
+    background: $qld-dark-blue;
+    color: $qld-white;
+    height: 72px;
+    left: 0;
+    bottom: 0;
+    padding: 16px;
+    position: fixed;
+    width: 100%;
+    z-index: 200;
+
+    .qld-quick-exit-inner {
```

Comment:
Looks good, only noticed one minor improvement with nested classes:

.qld-quick-exit { 
   &-inner //would work for (.qld-quick-exit-inner)
 }
 
 I usually do this last myself